ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -257,101 +258,46 @@ pattern CardanoNodeToNodeVersion1 :: BlockNodeToNodeVersion (CardanoBlock c)
 pattern CardanoNodeToNodeVersion1 =
     HardForkNodeToNodeDisabled ByronNodeToNodeVersion1
 
--- | The hard fork enabled with the latest Byron version and the Shelley era
--- enabled.
+-- | The hard fork enabled using the latest version of Byron and Shelley for
+-- each Byron and Shelley era.
 pattern CardanoNodeToNodeVersion2 :: BlockNodeToNodeVersion (CardanoBlock c)
 pattern CardanoNodeToNodeVersion2 =
     HardForkNodeToNodeEnabled
       HardForkSpecificNodeToNodeVersion1
-      (  EraNodeToNodeEnabled ByronNodeToNodeVersion2
-      :* EraNodeToNodeEnabled ShelleyNodeToNodeVersion1
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
+      (  WrapNodeToNodeVersion ByronNodeToNodeVersion2
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
+      :* WrapNodeToNodeVersion ShelleyNodeToNodeVersion1
       :* Nil
       )
 
--- | The hard fork enabled with the latest Byron version, the Shelley and
--- Allegra eras enabled.
+-- | A synonym for 'CardanoNodeToNodeVersion2' due to historical accident
 pattern CardanoNodeToNodeVersion3 :: BlockNodeToNodeVersion (CardanoBlock c)
-pattern CardanoNodeToNodeVersion3 =
-    HardForkNodeToNodeEnabled
-      HardForkSpecificNodeToNodeVersion1
-      (  EraNodeToNodeEnabled ByronNodeToNodeVersion2
-      :* EraNodeToNodeEnabled ShelleyNodeToNodeVersion1
-      :* EraNodeToNodeEnabled ShelleyNodeToNodeVersion1
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* EraNodeToNodeDisabled
-      :* Nil
-      )
+{-# DEPRECATED CardanoNodeToNodeVersion3 "Use the equivalent CardanoNodeToNodeVersion2 instead" #-}
```

Comment:
There is no occurrence of `CardanoNodeToNode` in cardano-api, cardano-cli or cardano-node, to I think it would also be fine to skip the deprecation cycle here for simplicity, especially as it is mentioned in the changelog.

Reply:
Done, thanks! Yeah, seems fair.